### PR TITLE
feat: add styling for informative audits

### PR DESCRIPTION
**Pull request by patrickhulce**
paving the way for #2112, #2044, and splitting up the byte efficiency audits into the sparkline versions and the more aggressive "did you know" versions

basically informative audits will always look like a blue information icon and will not look like pass/fails, the score now simply controls visibility and auto-openness

image

### CODE REVIEW

**Comment by ebidel** [comment]
Any reason not to use the existing iconography?

**Comment by patrickhulce** [comment]
> Any reason not to use the existing iconography?

Centering was just always off and it looked strange that large, it felt like neither would live until I/O so I didn't spend much time fiddling, but I'll throw back in there if there's a strong opinion

**Comment by ebidel** [comment]
Probably best to keep what we have until there's new direction from @chowse

**Comment by patrickhulce** [comment]
done
image

**Comment by ebidel** [comment]
LGTM. Can you add a quick test that checks that the class gets added?

**Comment by patrickhulce** [comment]
done

File: lighthouse-core/report/v2/renderer/category-renderer.js [hunk 1]
```diff
@@ -61,6 +61,11 @@ class CategoryRenderer {
       header.appendChild(this._detailsRenderer.render(audit.result.details));
     }
 
+    const scoreEl = this._dom.find('.lh-score', tmpl);
+    if (audit.result.informative) {
```
Comment (brendankenny) [inline lighthouse-core/report/v2/renderer/category-renderer.js]: I believe you'll need to add this to the `ReportRenderer.AuditJSON` typedef

Comment (patrickhulce) [inline lighthouse-core/report/v2/renderer/category-renderer.js]: done

File: lighthouse-core/test/report/v2/renderer/category-renderer-test.js [hunk 2]
```diff
@@ -75,6 +75,15 @@ describe('CategoryRenderer', () => {
     assert.ok(!audit2.querySelector('.lh-debug'));
   });
 
+  it('renders an informative audit', () => {
+    const auditDOM = renderer._renderAudit({
+      id: 'informative', score: 0,
+      result: {description: 'It informs', helpText: '', informative: true},
```
Comment (ebidel) [inline lighthouse-core/test/report/v2/renderer/category-renderer-test.js]: #TheInformator

**Pull request merged**